Assets/Samples/ProjectWideActions/ProjectWideActionsExample.cs

```diff
@@ -31,12 +31,6 @@ void Start()
                 previous = InputSystem.actions.FindAction("Player/Previous");
                 sprint = InputSystem.actions.FindAction("Player/Sprint");
                 crouch = InputSystem.actions.FindAction("Player/Crouch");
-
```

Comment:
Do we have a test for this anywhere seems like it would be useful to confirm. But don't block the PR for it.

Reply:
Good comment. What I would want is a Monobehavior test verifying the enable state in each possible Awake, Start, OnEnable etc. Haven't found a good way to achieve it fully yet. At least we test they are enabled in play mode for now. Not very specific of when. Will file a ticket if I cannot add such a test today.

Reply:
I have now found at least something in-between ideal testability and achievable testability using InputSystem internals. Will however postpone that to post this PR since test-only task. (ideally)